lib/usd/ui/demo/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+#
+# =======================================================================
+# Copyright 2019 Autodesk, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =======================================================================
+#
+
+cmake_minimum_required(VERSION 3.1.1)
+project(testMayaUsdUI)
+
+if (POLICY CMP0074)
+    cmake_policy(SET CMP0074 OLD)
+endif()
+
+add_executable(${PROJECT_NAME}
+	testMayaUsdUI.cpp
+)
```

Comment:
Simple text application to open the dialog. So I can test as standalone without Maya.